Classes/CharacterManager.gd

```diff
@@ -0,0 +1,115 @@
+class_name CharacterManager
+const script_name := "character_manager"
+
+static func create(character):
+	return CharacterDefaults.new().characters[character]
+
```

Comment:
Add a function to learn attacks instead of directly adding attacks to an array

Reply:
Also add a function that calls inventory.add and inventory.remove instead of directly accessing it to make things handled the same